src/config/contributors.json

```diff
@@ -735,10 +735,14 @@
       "2022": [
         "authors",
         "reviewers"
+      ],
+      "2024": [
+        "reviewers"
```

Comment:
Can't include this yet as you don't existing as a reviewer in any of the submitted chapters.

```suggestion
```